packages/shared/lib/core/router/enums/app-route.enum.ts

```diff
@@ -2,4 +2,5 @@ export enum AppRoute {
     Dashboard = 'dashboard',
     Login = 'login',
     Onboarding = 'onboarding',
+    Account = 'account',
```

Comment:
I think the name may be more precise. What do you think of `implicitAccountCreation`?